src/utils/formatSearchResponse.ts

```diff
@@ -73,13 +144,13 @@ const getOtherDetails = async (searchItem: Record<string, any>): Promise<Record<
     metadata_standard: searchItem?._source?.standardNameObject?.default ?? '',
     project_number: '',
     Metadata_language: searchItem?._source?.mainLanguage ?? '',
-    ncea_catalogue_date: catalogueDate
-      ? catalogueDate.toLocaleDateString('en-US', { day: 'numeric' }) +
-        '-' +
-        catalogueDate.toLocaleDateString('en-US', { month: 'short' }) +
-        '-' +
-        catalogueDate.toLocaleDateString('en-US', { year: 'numeric' })
-      : '',
+    ncea_catalogue_date: getCatalogueDate(searchItem),
```

Comment:
I believe we have done this to fix the function complexity suggested by Sonar. But we still have a few more tabs. After we add all the necessary things, this error might occur again. So maybe we can move this logic to another file and import it here.
With this, we can also write test scenarios for each tab of data that we receive and return.
Just for FYI, I have this feeling about the complexity issue and did similar things in my open [PR](https://github.com/DEFRA/ncea-frontend/blob/2dd07ec8d60836af5b77bd3f40a9b7f7f4a9571b/src/utils/formatSearchResponse.ts#L90).